react/testing_4/src/app/screens/Home/components/FoodList/components/Food/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import "./styles.css";
+
+function Food({ food }) {
```

Comment:
I'd receive directly `name, description, imageUrl` as props. What do you think ?